test/test-template-service.ts

```diff
@@ -73,13 +73,18 @@ describe("TemplateService Api", () => {
             templateService.getTemplates().then((templates: any) => {
                 should.exist(templates);
                 templates.should.be.an("array");
+                templates.length.should.be.above(0);
```

Comment:
Maybe it is a good idea to check the length with the availableTemplateRepos array length in the /shared/config file.

E.g: 
import { Config } from "../lib/shared/config";
expect(templates).to.have.length(Config.availableTemplateRepos.length);

Reply:
if the getAppTemplateDetails() promise in template-service fails, will the array contain anything?